tests/gocase/unit/type/list/list_test.go

```diff
@@ -1000,4 +1000,100 @@ func TestList(t *testing.T) {
 			require.Empty(t, rdb.LPosCount(ctx, "mylist", "b", 10, redis.LPosArgs{Rank: 5}).Val())
 		})
 	}
+
```

Comment:
```go
// LMPop Pops one or more elements from the first non-empty list key from the list of provided key names.
// direction: left or right, count: > 0
// example: client.LMPop(ctx, "left", 3, "key1", "key2")
func (c cmdable) LMPop(ctx context.Context, direction string, count int64, keys ...string) *KeyValuesCmd {
	args := make([]interface{}, 2+len(keys), 5+len(keys))
	args[0] = "lmpop"
	args[1] = len(keys)
	for i, key := range keys {
		args[2+i] = key
	}
	args = append(args, strings.ToLower(direction), "count", count)
	cmd := NewKeyValuesCmd(ctx, args...)
	_ = c(ctx, cmd)
	return cmd
}
```

Seems `go-redis` `LMPOP` will force a `COUNT` argument? should we try to test without `COUNT`?

Reply:
Sure, I am not familiar with go-redis, so have no idea how to strip the 'count' argument. But now it occurs that I could simply copy-and-modify the go-redis's LMPOP function... Will do it, thank you.

Reply:
@HolyLow Can use the go-redis client's Do function which can pass the command arguments instead of encapsulating one.

Reply:
you can do this (just issue the command with the plain Do):
```go
require.NoError(t, rdb.Do(ctx, "LMPOP", "1", "mykey", "left").Err())
```

Reply:
The LMPOP test cases with no COUNT keyword are added now.